src/app/core/components/info-card/info-card.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="card">
+  <div class="message-container">
+    <p>{{ label }}</p>
```

Comment:
Should it be `<p>`? Consider span or no tag because you have div

Reply:
Fix it: 9c3204c8a2f7ea6f152e54eb9d5da92ef6f3f554